code/builders/builder-vite/src/vite-config.ts

```diff
@@ -77,6 +77,12 @@ export async function commonConfig(
 
 export async function pluginConfig(options: Options) {
   const frameworkName = await getFrameworkName(options);
+  const { features } = options;
+  const globalsObj: Record<string, string> = globals;
+
+  if (features?.fastBuildEmptyBlocks) {
+    globalsObj['@storybook/blocks'] = '__STORYBOOK_BLOCKS_EMPTY_MODULE__';
```

Comment:
I just made something up... This won't actually exist.!

If it not existing is a problem, we could chose either:
1. Just accept it, users can opt-out of this featureFlag using `main.ts`
2. We actually inject a global (we can do this from the builders, it's fairly straight forwards), possibly even with a dummy interface so everything exported can actually be found (but would no-op).

Right now, I'm leaning towards "let's do nothing, and see what happens.".